nengo/builder/operator.py

```diff
@@ -247,45 +247,87 @@ def step_copy():
 
 
 class SlicedCopy(Operator):
-    """Copy from `a` to `b` with slicing: `b[b_slice] = a[a_slice]`"""
-    def __init__(self, a, b, a_slice=Ellipsis, b_slice=Ellipsis,
+    """Assign the value of a slice of one signal to another slice.
+
+    Implements ``dst[dst_slice] = src[src_slice]``.
+
+    This operator can also implement ``dst[dst_slice] += src[src_slice]``
+    using the parameter ``inc``.
+
+    Parameters
+    ----------
+    dst : Signal
+        The signal that will be assigned to (set).
+    src : Signal
+        The signal that will be copied (read).
+    dst_slice : slice or Ellipsis, optional (Default: Ellipsis)
+        Slice associated with ``dst``.
+    src_slice : slice or Ellipsis, optional (Default: Ellipsis)
+        Slice associated with ``src``
+    inc : bool, optional (Default: False)
+        Whether this should be an increment rather than a copy.
+    tag : str, optional (Default: None)
+        A label associated with the operator, for debugging purposes.
+
+    Attributes
+    ----------
+    dst : Signal
+        The signal that will be assigned to (set).
+    dst_slice : list or Ellipsis
+        Indices associated with ``dst``.
+    src : Signal
+        The signal that will be copied (read).
+    src_slice : list or Ellipsis
+        Indices associated with ``src``
+    tag : str or None
+        A label associated with the operator, for debugging purposes.
+
+    Notes
+    -----
+    1. sets ``[] if inc else [dst]``
+    2. incs ``[dst] if inc else []``
+    3. reads ``[src]``
+    4. updates ``[]``
+    """
+
+    def __init__(self, dst, src, dst_slice=Ellipsis, src_slice=Ellipsis,
```

Comment:
I would prefer `src, dst`, and change `Copy` to be the same. That seems more similar to me to things like `DotInc`, `SimProcess`, etc., with the inputs first then the outputs.